XMonad/Hooks/WallpaperSetter.hs

```diff
@@ -100,9 +100,16 @@ defWallpaperConf = WallpaperConf "" $ WallpaperList []
 instance Default WallpaperConf where
     def = defWallpaperConf
 
--- |returns the default association list (maps name to name.jpg, non-alphanumeric characters are omitted)
 defWPNames :: [WorkspaceId] -> WallpaperList
-defWPNames xs = WallpaperList $ map (\x -> (x,WallpaperFix (filter isAlphaNum x++".jpg"))) xs
+defWPNames = defWPNamesJpg
+
+-- |returns the default association list (maps name to name.jpg, non-alphanumeric characters are omitted)
+defWPNamesJpg :: [WorkspaceId] -> WallpaperList
+defWPNamesJpg xs = WallpaperList $ map (\x -> (x, WallpaperFix (filter isAlphaNum x ++ ".jpg"))) xs
+
+-- |returns the default association list (maps name to name.png, non-alphanumeric characters are omitted)
+defWPNamesPng :: [WorkspaceId] -> WallpaperList
+defWPNamesPng xs = WallpaperList $ map (\x -> (x, WallpaperFix (filter isAlphaNum x ++ ".png"))) xs
```

Comment:
Perhaps instead of replicating almost the same documentation, highlight the differences instead.

For example,

``` haskell
-- | Like `defWPNamesJpg`, but map @name@ to @name.png@ instead.
```